### ISSUE TITLE
Demo.py failed

### ISSUE THREAD

**Issue by Dxye** [smallcorgi/Faster-RCNN_TF#103]
Dear guys,
I ran demo.py and met the ModuleNotFoundError: No module named 'roi_pooling_op'.
My OS is ubuntu16.04LTS.
Anyone can help me?

**Comment by qinglintian** [comment]
Did you followed the instructions on readme? I think you missed the "make" procedure in lib directory

**Comment by neufratel** [comment]
Try to chane line 4 in make.sh to:
 CXXFLAGS='-D_MWAITXINTRIN_H_INCLUDED'

**Comment by Dxye** [comment]
Yes, I ran 'make' in lib. I solved it by add 'roi_pooling_layer.' before 'roi_pooling_op'
thank you @qinglintian

**Comment by a3551571** [comment]
@Dxye 
I meet the same problem with you.
Could you show me more details about how to solve this problem?
Thanks!

**Comment by Dxye** [comment]
@a3551571 sorry but i cannot remember such accurate, try add 'import roi_pooling_layer' after 5th line in demo.py

**Comment by a3551571** [comment]
@Dxye 
OK,Thanks a lot!

**Comment by longzeyilang** [comment]
@Dxye I met the problem. Could you show me more details about how to solve this problem?

**Comment by guigui10011** [comment]
@longzeyilang I solved this problem by adding 'import roi_pooling_layer.roi_pooling_op' instead of 'import roi_pooling_op' in the file roi_pooling_op_grad.py (that was the one giving me the error).
